lib/proxyquire.js

```diff
@@ -136,17 +137,45 @@ Proxyquire.prototype.load = function (request, stubs) {
   return this._withoutCache(this._parent, stubs, request, this._parent.require.bind(this._parent, request));
 };
 
+// Resolves a stub relative to a module.  Note that this only resolves
+// "relative" paths (e.g. "./foo") but not non-relative modules (e.g. "react").
+// Also, if noPreserveCache(), then we won't throw if the module can't
+// be resolved.
+Proxyquire.prototype._resolveRelativePath = function(path, moduleFilename) {
+  try {
+    if(path[0] === '.') {
```

Comment:
Perhaps I'm overthinking this, and we should just resolve everything, instead of only the relative paths.  Then we wouldn't have to resolve them all a second time in the cleanup part of _withoutCache()...  Yeah, I think I'm going to make that change.

Reply:
Except, when I try this it breaks a bunch of unit tests.  Maybe I'll leave it alone.  :)

Reply:
`if (`